roundtables/day05-reverse-string-portfolio-site.md

```diff
@@ -0,0 +1,31 @@
+# Exercism: Reverse String & Portfolio Site "Show and Tell"
+
+Today we are going to be discussing the Reverse String exercise and doing a "show and tell" of our portfolio sites. As a reminder, your portfolio site is not expected to be perfect nor do we expect them to look like they were created by a professional team of web developers. This is just a fun way for everyone to show off what you were able to make in a few hours of creating.
+
+For this activity, you are encouraged to do the following:
+
+* Whenever possible, you should use voice.
+* You should pose questions to the group, not the instructor.
+* Similarly, you are encouraged to answer questions, with the instructor available to correct misconceptions and provide answers when no one else has an answer.
+* Say something, even if you think you might be wrong! (Nobody throws a frisbee well on their first try)
+* When discussing code, you are encouraged to use VS Code Liveshare, and or screen sharing through zoom.
+* Take notes!
+
+## Materials
+[Exercism (JavaScript Track)](https://exercism.io/my/tracks/javascript)
+
+[Personal Portfolio Project](https://github.com/Ada-C13/personal-portfolio-site)
+
+
+## Prompts
+You are encouraged to copy these questions into a document and take notes during the discussion.
+
+1. What questions did you have from completing the Reverse String exercise? Was there anything particularly challenging? (Try to keep this to 10 minutes if possible to give plenty of time for show-and-tell time.)
+    * Let's view 2 different implementations! Volunteers?
+1. Show and Tell!
+    * Take turns showing off your Personal Portfolio Site! 
+    * If it helps, here are some prompts for what to talk about
+        * What is one thing you did for your site you're proud of?
+        * What was a challenge you faced while working on the site?
+        * What was your workflow of working on the skeleton of the site (HTML) and the styling (CSS)? Did you refine that workflow at all over the course of your time working on the site?
+    * Everyone should get a chance to share! So please let your presenting time be 3-5 minutes.
```

Comment:
It would be better for structuring things if this was broken into multiple smaller pieces just to make it easier to move through the kind of big "question 1", "question 2", etc.

Reply:
I'm not sure how to break it up any further. Can you explain how you're envisioning breaking them up into smaller pieces?

Reply:
I'm not sure how to break it up any further. Can you explain how you're envisioning breaking them up into smaller pieces?